### DISCUSSION TITLE
Image | ASUS Tinker Board 2

### DISCUSSION THREAD

**Discussion by provalinf** [MichaIng/DietPi#5023] [Image Request]
Hi,
I propose this SBC which is still based on an RK3399 (again), on which dietpi would be useful to me personally, but also because it seems to be neglected by other distributions apart from tinkerOS (Debian 10) proposed by Asus which is of enormous size 4Go.
I had tried to run the PREP_SYSTEM_FOR_DIETPI script but it was not very successful on reboot.

#### Formal device information
- Device name | Tinkerboard 2 & (2S <- with EMMC)
- Official product URL | https://tinker-board.asus.com/product/tinker-board-2.html
- GitHub resource URL (if available) | https://github.com/TinkerBoard2/
- Image download URL | https://tinker-board.asus.com/download-list.html?product=tinker-board-2 (Debian 10 & Android 10/11)

Thx

**Comment by Joulinar** [comment]
Many thanks for your request. I moved it to discussions where we start to collect image requests now.

**Comment by atweek** [comment]
Is there any hope for an addition?

**Comment by ronatzu** [comment]
Is there a trial version or something for this SBC?

**Reply by MichaIng** [reply]
Armbian is picking up support again for this board, so jep, I'll add support for next release: https://www.armbian.com/tinkerboard-2/

**Reply by vanjagrozniy** [reply]
Great news! Did you manage to get it work?

**Comment by MichaIng** [comment]
A testing image is available here: https://dietpi.com/downloads/images/testing/
Would be great is someone could give is a try.

**Comment by MichaIng** [comment]
All done, images are available on our download page now.
